cluster/ubuntu/build.sh

```diff
@@ -26,12 +26,15 @@ function cleanup {
 }
 trap cleanup SIGHUP SIGINT SIGTERM
 
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+pushd ${KUBE_ROOT}/cluster/ubuntu
```

Comment:
I could be misreading, is there any problem with this? 
`pushd` here manages to make sure that the binaries dir locates expectedly.